tools/rpmkeys.cc

```diff
@@ -149,7 +155,8 @@ int main(int argc, char *argv[])
 
     args = (ARGV_const_t) poptGetArgs(optCon);
 
-    if (args == NULL && mode != MODE_LISTKEY && mode != MODE_EXPORTKEY)
+    if (args == NULL && mode != MODE_LISTKEY && mode != MODE_EXPORTKEY &&
+	mode != MODE_REBUILD)
```

Comment:
Indent nitpicks: this is confusingly on the same indent level as the codeblock below.

Reply:
If I only could remember how to indent this properly...

Reply:
For crying out loud. It's not like you need to remember 21 digits of pi here. The issue is *ambiguity* due to identical indentation so to fix you just need to change how it ends up laid out, such as:

a) indent the condition continuation deeper.
b) put the opening brace on the next line (unlike the usual style), this provides the brain sufficient clue about the matter
c) avoid the whole issue by moving part of the condition elsewhere, such as using an aptly named helper variable

Quite often it's c) that makes for most readable code.

Reply:
OK, so the reason is that there are too many, all not so great options. Went with a) already.

Reply:
Yeah a) and b) are mechanically applicable anywhere, c) requires thinking.

As a reminder, CONTRIBUTING.md has advice too:
> If in doubt, 'indent -kr' generally produces acceptable formatting.